### PR TITLE
[GH Issue Templates] Replaced ng with master

### LINKED ISSUE #1602: [GH Issue Templates] Replace layer5-ng references with master
**Description**
This repo's GH issue templates still have references to the `layer5-ng` branch.

**Expected Behavior**
Should reference `master`

### PR DESCRIPTION
**Pull request by EmperorYP7**
Signed-off-by: Yash Pandey (YP) <yash.btech.cs19@iiitranchi.ac.in>

**Description**

This PR fixes #1602 

**Notes for Reviewers**
NA

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/events.md [hunk 1]
```diff
@@ -16,5 +16,6 @@ assignees: ''
 ---
 **Contributor Resources**
 
-The layer5.io website uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
-- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
+The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
+- See [`master` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
```
Comment (leecalcote) [inline .github/ISSUE_TEMPLATE/events.md:20]: I should have noted that e don't need to specify master here. We can just say, "see contributing instructions" without having to specify master.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -35,5 +35,6 @@ Steps to reproduce the behavior:
 ---
 **Contributor Resources**
 
-The layer5.io website uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
-- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
+The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
+- See [`master` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
```
Comment (leecalcote) [inline .github/ISSUE_TEMPLATE/bug_report.md:39]:
```suggestion
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
```

File: .github/ISSUE_TEMPLATE/chore_task.md [hunk 3]
```diff
@@ -21,6 +21,6 @@ assignees: ''
 ---
 **Contributor Resources**
 
-The layer5.io website uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
-- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
-
+The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
+- See [`master` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
```
Comment (leecalcote) [inline .github/ISSUE_TEMPLATE/chore_task.md:25]:
```suggestion
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
```

File: .github/ISSUE_TEMPLATE/community_member_profile.md [hunk 4]
```diff
@@ -22,4 +22,6 @@ Let's recognizing <@github-username> as a contributor and community member by cr
 
 **Contributor Resources**
 
-The layer5.io website, which uses GatsbyJS and GitHub Pages. Site content is found under the <a href="https://github.com/layer5io/layer5/tree/master">`master` branch</a>. See <a href="https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md">`layer5` contributing instructions</a>.
+The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
+- See [`master` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
```
Comment (leecalcote) [inline .github/ISSUE_TEMPLATE/community_member_profile.md:26]:
```suggestion
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 5]
```diff
@@ -21,10 +21,6 @@ assignees: ''
...
 **Contributor Resources**
 
-The layer5.io website uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
-- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
-
-
-Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
-- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
+The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
+- See [`master` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
```
Comment (leecalcote) [inline .github/ISSUE_TEMPLATE/feature_request.md:25]:
```suggestion
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
```

File: .github/ISSUE_TEMPLATE/landscape.md [hunk 6]
```diff
@@ -11,7 +11,7 @@ assignees: ''
...
 ---
 **Contributor Resources**
-- [Instructions for updating the landscape](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#updating-the-service-mesh-landscape)Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
-
-The layer5.io website uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
-- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
\ No newline at end of file
+- [Instructions for updating the landscape](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#updating-the-service-mesh-landscape)
+The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
+- See [`master` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
```
Comment (leecalcote) [inline .github/ISSUE_TEMPLATE/landscape.md:16]:
```suggestion
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
```

File: .github/ISSUE_TEMPLATE/engagement.md [hunk 7]
```diff
@@ -13,5 +13,7 @@ assignees: ''
 
 ---
 **Contributor Resources**
-The layer5.io website uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
-- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
+
+The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
+- See [`master` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
```
Comment (leecalcote) [inline .github/ISSUE_TEMPLATE/engagement.md:18]:
```suggestion
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
```

File: .github/ISSUE_TEMPLATE/news.md [hunk 8]
```diff
@@ -13,9 +13,7 @@ We're in the news! Using the contributing instructions for [adding news](https:/
 
 ---
 **Contributor Resources**
-The layer5.io website uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
-- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
 
-Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
-- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
+The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
+- See [`master` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
```
Comment (leecalcote) [inline .github/ISSUE_TEMPLATE/news.md:18]:
```suggestion
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
```

**Review by leecalcote — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/events.md (hunk 1 above)
Comment (leecalcote) [inline .github/ISSUE_TEMPLATE/events.md:20]:
```suggestion
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
```

**Review by leecalcote — APPROVED** [review]

**Comment by EmperorYP7** [comment]
Oops! Missed it.. 😓

**Comment by leecalcote** [comment]
No sweat. Teamwork, baby, teamwork.

**Pull request merged**